### DISCUSSION TITLE
Area series change color based on time

### DISCUSSION THREAD

**Discussion by dmyerscough** [tradingview/lightweight-charts#1242] [Q&A] [answered]
Is it possible to change the color and gradient of an AreaSeries based on the time?

**Comment by SlicedSilver** [comment, marked as answer]
This is now available in [v4.0.0](https://github.com/tradingview/lightweight-charts/releases/tag/v4.0.0)

A similar approach on how to achieve this is shown here: https://tradingview.github.io/lightweight-charts/tutorials/customization/data-points

Related issue: https://github.com/tradingview/lightweight-charts/issues/1100
